### DISCUSSION TITLE
Access denied on public package pulled by docker

### DISCUSSION THREAD

**Discussion by korthout** [community/community#50487] [Packages] [answered]
### Select Topic Area

Bug

### Body

The documentation on [package visibility](https://docs.github.com/en/packages/learn-github-packages/configuring-a-packages-access-control-and-visibility#configuring-visibility-of-packages-for-your-personal-account) is quite clear:

> A public package can be accessed anonymously without authentication.

I cannot access my public container package anonymously.

```sh
$ docker logout
Removing login credentials for https://index.docker.io/v1/

$ docker pull ghcr.io/korthout/camunda-platform-rest-api:latest
Error response from daemon: Head "https://ghcr.io/v2/korthout/camunda-platform-rest-api/manifests/latest": denied: denied
```

What am I missing?

- The [package](https://github.com/korthout/camunda-platform-rest-api/pkgs/container/camunda-platform-rest-api) is set to `public` visibility
- The related repository ([korthout/camunda-platform-rest-api](https://github.com/korthout/camunda-platform-rest-api)) has `public` visibility as well
- I've tried to disable/enable the package's setting to `Inherit access from source repository (recommended)`, but the result is the same

Any help is appreciated, but this appears like a bug to me.

**Comment by krystian-jedrzejowski-zazoon** [comment]
I have the same issue

**Comment by jibbolo** [comment]
The same is happening to me

```
~ ➤ docker logout
Removing login credentials for https://index.docker.io/v1/

~ ➤ docker pull ghcr.io/jibbolo/cwdash:latest
Error response from daemon: Head "https://ghcr.io/v2/jibbolo/cwdash/manifests/latest": denied: denied
```

**Comment by korthout** [comment]
I want to add that this is blocking some users from getting started, and I wonder if I should move the container image over to another registry. I prefer to leave it on ghcr.io, but we're nearing two weeks without an official reply. It would be nice to know if anyone from GitHub is looking into this.

**Comment by airtower-luna** [comment, marked as answer]
If you use `docker logout` without a domain it logs you out from Docker Hub, not GHCR. So if you have a credential for `ghcr.io` it'll stay. If you want to test anonymous access to your image, you need to use `docker logout ghcr.io`.

**Reply by korthout** [reply]
Thanks ❤️ This resolves the issue!

**Reply by itonyr** [reply]
+1 this helped me as well years later :)
